features/restaurant_grouping.feature

```diff
@@ -0,0 +1,19 @@
+Feature: User can select restaurant by category
+  As a user
+  In order to get choose what to eat
+  I would like to see a restaurant by category
+
+  Background:
+    Given the following category exists
+      | name    | description  |
+      | Thai    | Thai food    |
+      | Mexican | Mexican food |
+    And the following restaurants exist
+      | name       | address     | restaurant_category_id |
+      | My Thai    | Stockholm   | 1                      |
+      | My Mexican | Mexico city | 2                      |
```

Comment:
Much better but I would avoid using the restaurant if. You know what name it has.  Search for it by name